src/Mailer/Email.php

```diff
@@ -827,6 +835,29 @@ public function headerCharset($charset = null)
         return $this->headerCharset;
     }
 
+    /**
+     * TransferEncoding setter.
+     *
+     * @param string|null $encoding Encoding set.
+     * @return $this
+     */
+    public function setTransferEncoding($encoding)
+    {
+        $this->transferEncoding = $encoding;
```

Comment:
Should we validate that for a valid/known transfer encoding?

Reply:
I don't know all content transfer encoding. But I found [this](https://www.w3.org/Protocols/rfc1341/5_Content-Transfer-Encoding.html). 
I understand that there is `BASE64 / QUOTED-PRINTABLE / 8BIT / 7BIT / BINARY / x-token` but I'm not sure, I'm not a expert.

If you think is good, I can implement it.

Reply:
I think checking the standard types (not `x-token`) is a good place to start.